generators/server/files/page.js

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'crypto';
+
+function torbit(markup) {
+  return createHash('sha256').update(markup).digest('hex');
+}
```

Comment:
I like this approach; there are two things we can do with it. Firstly the `etag` function can be overridden:

```javascript
app.set('etag', (body, encoding) => { ... });
```

And secondly the `eTag` for the request itself can be set to prevent the default from being generated:

```javascript
app.use((req, res, next) => {
  res.set('ETag', 'somevalue');
});
```

The first approach will work for the use case at BigRetailer™ since they need hashing for PCI compliant pages; the second may be interesting to look at regarding `redux` – could make a hash based on the state that `redux` is given.